samples/poc-subappv1-csp/src/routes.js

```diff
@@ -30,7 +48,8 @@ export default {
     // Enable one of these to use CSP header
     cspNonce: true,
```

Comment:
need to disable this for this sample since app code is passing down header info

Reply:
cspNonce value would be required, as if user wants to set script-src or style-src, and doesnot set the nonce, we'd generate and pass as params in `cspHeaderValues` in routes. This is to keep the as-is behaviour.

Reply:
ah ok, my bad. nvm